src/Utilities/PaymentUtils.res

```diff
@@ -8,8 +8,12 @@ let paymentListLookupNew = (
   ~isKlarnaSDKFlow,
   ~paymentMethodListValue: PaymentMethodsRecord.paymentMethodList,
   ~areAllGooglePayRequiredFieldsPrefilled,
+  ~areAllPaypalRequiredFieldsPreFilled,
   ~isGooglePayReady,
   ~shouldDisplayApplePayInTabs,
+  ~isPaypalSDKFlow,
+  ~isPaypalRedirectFlow,
```

Comment:
Instead of prop drilling all of these fields, could you please calculate the boolean value before and just pass down one prop?

Reply:
Updated